### DISCUSSION TITLE
Bug Report Template

### DISCUSSION THREAD

**Discussion by pcs289** [pheature-flags/pheature-flags#308] [Q&A] [answered]
We would need to decide what to do on the templates for bug reports.

The main idea is to have:

- Base Template for all repositories (thanks-banner, description, steps-reproduce, relevant-output, consent-tc)
- Specific sections for some repositories (integration section for pheature-flags monorepo vs component section for toggle-ui)

Currently we have three versions:
- [bug.yml](https://github.com/pheature-flags/pheature-flags/blob/1.0.x/.github/ISSUE_TEMPLATE/bug.yml)
- [bug_report.yaml](https://github.com/pheature-flags/toggle-ui/pull/3)
- [bug_report.yml](https://github.com/pheature-flags/toggle-core/pull/26)

Before deciding upon any version, it would be awesome to discuss a bit further, not only for choosing one option but to find the best approach for bug templates.

**Comment by kpicaza** [comment]
Take a look at this repo https://github.com/stevemao/github-issue-templates we can get some ideas from there ;-D

**Comment by pcs289** [comment, marked as answer]
I think we should go for [bug.yml](https://github.com/pheature-flags/pheature-flags/blob/1.0.x/.github/ISSUE_TEMPLATE/bug.yml) (which contains a Packages Dropdown) for the following packages:
- PSR11-Factories
- PSR7-API
- Toggle-CRUD
- InMemory
- DBal

On the rest of the packages, I'd go for [bug_report.yaml](https://github.com/pheature-flags/toggle-ui/pull/3/files) as it includes all maintainers:
- Toggle-Model
- Toggle-Core
-  Mezzio
- Laravel
- Symfony
- PHP SDK
